spec/integration/Cursor_with_resultset_handle_spec.lua

```diff
@@ -63,17 +65,24 @@ describe("Cursor with resultset handle", function()
         connection:close()
     end
 
-    it("fetches small result sets with default fetchsize", function() test_result_with(999) end)
+    it("fetches small result sets with default fetchsize", function()
+        test_result_with(999)
+    end)
 
     it("fetches large result sets with default fetchsize [itest -> dsn~luasql-cursor-fetch-resultsethandle~0]",
-       function() test_result_with(2000) end)
+       function()
+        test_result_with(2000)
```

Comment:
Here the indentation looks off, probably because the formatter aligns `function() with the previous parameter in `it`. This is not a really issue though. Just looks a little strange.